Tools/MiniBrowser/playstation/MainWindow.cpp

```diff
@@ -106,7 +109,27 @@ MainWindow::MainWindow(const char* requestedURL)
     }});
 
     createNewWebView(nullptr);
-    activeWebView()->loadURL(requestedURL ? requestedURL : "https://webkit.org");
+
+    // Parse command line options
+    std::string requestedURL = "";
+
+    parseOptions(options, &requestedURL);
+
+    activeWebView()->loadURL(!requestedURL.empty() ? requestedURL.c_str() : "https://webkit.org");
+}
+
+void MainWindow::parseOptions(const std::vector<std::string>& options, std::string* requestedURL)
```

Comment:
This `requestedURL` should never be nullptr because you dereference it unconditionally. In such case, you should use "std::string&".